src/libs/actions/Report.js

```diff
@@ -168,6 +168,53 @@ function getChatReportName(fullReport, chatType) {
         .join(', ');
 }
 
+/**
+ * Returns number of optimisticReportActionIDs exist in reportActions
+ *
+ * @param {String} reportID
+ * @returns {Number}
+ */
+function getNumberOfOptimisticActions(reportID) {
+    const optimisticActionIds = lodashGet(allReports, [reportID, 'optimisticReportActionIDs'], []);
+    return _.filter(optimisticActionIds, actionId => ReportActions.isOptimisticReportActionExist(reportID, actionId)).length;
+}
+
+/**
+ * If onyx local report data is newer than fetched report then return updated version of simplifiedReport
+ * Local report data could be newer than fetched report because pusher new comment event coming while fetch report request isn't completed yet
```

Comment:
```suggestion
 * Local report data could be newer than the fetched report because of a pusher new comment event coming while the fetching report request hasn't been completed yet
```

Reply:
My english is bad. Thank you for the review.